src/shared/utils/reactProdInvariant.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule reactProdInvariant
```

Comment:
The pattern is actually this:
- export an object or class? Upper case
- export a function? lower case

The only other one we have of the latter that starts with `react` is `reactComponentExpect`